src/OrleansManager/Program.cs

```diff
@@ -166,7 +166,7 @@ private static void UnregisterGrain(string[] args)
 
             var directory = GrainClient.InternalGrainFactory.GetSystemTarget<IRemoteGrainDirectory>(Constants.DirectoryServiceId, silo);
 
-            WriteStatus(string.Format("**Calling DeleteGrain({0}, {1}, {2})", silo, grainId));
+            WriteStatus(string.Format("**Calling DeleteGrain({0}, {1})", silo, grainId));
             directory.DeleteGrainAsync(grainId).Wait();
```

Comment:
This PR looks OK, but I wonder if this method should be changed to an async one or `GetAwaiter().GetResult()` used instead, as has come up before.

@dVakulen knows this, but included to bring context for larger audience, more information [here](https://github.com/aspnet/Security/issues/59) and [here](http://blog.stephencleary.com/2014/12/a-tour-of-task-part-6-results.html).